kedro-datasets/kedro_datasets/api/api_dataset.py

```diff
@@ -57,14 +50,55 @@ class APIDataSet(AbstractDataSet[None, requests.Response]):
         >>>     credentials=("username", "password")
         >>> )
         >>> data = data_set.load()
+
+    ``APIDataSet`` can also be used to save some output on some remote server using
+    HTTP(S) methods.
+
+        >>> import pandas as pd
+        >>> example_table = pd.DataFrame({"col1":["val1", "val2"], "col2":["val3", "val4"]}
+
+    Here we initialise our APIDataSet with the correct parameters to make requests
+    towards the configured remote server.
+
+        >>> data_set = APIDataSet(
+                method = "POST"
+                url = "url_of_remote_server",
+                save_args = {"chunk_size":1}
+        )
+    On initialization, we can specify all the necessary parameters in the save args
```

Comment:
To keep it consistent with the GB spelling elsewhere 🇬🇧 

```suggestion
    On initialisation, we can specify all the necessary parameters in the save args
```